code/controllers/subsystem/ai_move_test.dm

```diff
@@ -0,0 +1,44 @@
+//Potential implementation of a tile by tile movement via subsystem
+
+SUBSYSTEM_DEF(ai_movement)
+	name = "ai tile by tile"
+	priority = FIRE_PRIORITY_DEFAULT
+	flags = SS_KEEP_TIMING
+	init_order = INIT_ORDER_AI_MOVE
+	wait = 0.5
```

Comment:
Not sure if this is ever gonna be valid or possible will always be 1 i suspect.

Reply:
1 means every 0.5 seconds.